src/pybind/mgr/cephadm/module.py

```diff
@@ -277,6 +277,7 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
             default='*',
             desc='PlacementSpec describing on which hosts to manage /etc/ceph/ceph.conf',
         ),
+        # not used anymore
```

Comment:
I think we should remove these options entirely.  If I'm reading the code properly, the migration ``config rm`` calls should still work.

Reply:
i wanted to do that but i was having trouble getting the old values during the migration.

once the new mgr takes over during an upgrade and with those removed
`registry_url = self.mgr.get_module_option('registry_url')`
causes
`RuntimeError: Config option 'registry_url' is not in CephadmOrchestrator.MODULE_OPTIONS`


and 
```
ret, registry_url, err = self.mgr.mon_command({
                'prefix': 'config get',
                'who': 'mgr',
                'key': 'mgr/cephadm/registry_url',
            })
```
returns a return code of `-2` and no value for registry_url


additionally trying to run 

`ceph config get mgr mgr/cephadm/registry_url`

returns `Error ENOENT:
`

Reply:
I think I should add a ``--force`` to ``config get``... WDYT @sebastian-philipp ?

Reply:
Not yet made my mind about this one. IMO this is an extraordinary edge case. Is it worth building extra code for this?